.luacheckrc

```diff
@@ -42,7 +42,7 @@ globals = { -- Globals
             "SaveGame", "SaveGameFile", "Staff", "StaffProfile", "StaffRoom",
             "Strings", "SwingDoor", "TheApp", "TreeControl", "Vip", "Window",
             "World", "Date", "Doctor", "Handyman", "Nurse", "Receptionist",
-            "Cheats",
+            "Cheats", "AnimationOverlayFlags",
```

Comment:
Why I was lazy and put "Cheats" at the end of this list is a bad move by myself.
Ideally we want to keep them in alphabetic order (but blame myself for this :) )

Reply:
I also blame lewri :p

Reply:
I can put alphabetically. I prefer to add chronologically by the time it was added to the code.

Reply:
I think my "Cheats" should be moved onto Line 35. Yours would go nicely before "App" on Line 34.